docs/src/conf.py

```diff
@@ -1,13 +1,25 @@
+import os
 from datetime import datetime
 
 
+ROOT = os.path.abspath(os.path.join("..", ".."))
+
 # Add any Sphinx extension module names here, as strings.
 extensions = [
     "sphinx.ext.viewcode",
     "sphinx.ext.intersphinx",
-    "sphinx_gallery.load_style",
+    "sphinx_gallery.gen_gallery",
+    "chemiscope.sphinx",
 ]
 
+examples_dirs = os.path.join(ROOT, "examples")
+sphinx_gallery_conf = {
+    "examples_dirs": examples_dirs,
+    "gallery_dirs": "examples",
+    "filename_pattern": ".*",
+    "within_subsection_order": "FileNameSortKey",
+}
+
```

Comment:
this is not needed, the configuration for rst generation is in the root `generate-gallery.py`